src/api/components/PostList/PostList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Post } from '../Post/Post';
+
+import './PostList.scss';
+
+export const PostList = ({ postList }) => (
+  <ul className="PostList">
+    {postList.map(post => (
+      <li key={post.id} className="PostList__item">
+        <Post {...post} />
+      </li>
+    ))}
+  </ul>
+);
+
+PostList.propTypes = {
+  postList: PropTypes.arrayOf(
+    PropTypes.shape({}),
```

Comment:
```suggestion
    PropTypes.shape({
      id: PropTypes.number.isRequired,
    }),
```
You use `id` here so it's better to describe it